include/zephyr/storage/flash_map.h

```diff
@@ -232,6 +253,20 @@ uint32_t flash_area_align(const struct flash_area *fa);
 int flash_area_get_sectors(int fa_id, uint32_t *count,
 			   struct flash_sector *sectors);
 
+/**
+ * Retrieve info about sectors within the area.
+ *
+ * @param[in]  fa       pointer to flash area object.
+ * @param[out] sectors  buffer for sectors data
+ * @param[in,out] count On input Capacity of @p sectors, on output number of
+ * sectors Retrieved.
```

Comment:
Nit: keep order as parameters?

```suggestion
 * @param[in]  fa       pointer to flash area object.
 * @param[out] sectors  buffer for sectors data
 * @param[in,out] count On input Capacity of @p sectors, on output number of
 * sectors retrieved.
```